mypy/plugin.py

```diff
@@ -1,5 +1,9 @@
 """Plugin system for extending mypy."""
 
+import sys
+import os.path
```

Comment:
It looks like these two imports are also unused. Is this also for mypyc or you just forgot to remove them?

Reply:
Oh, this is from a legacy of trying a few different things for the `if MYPYC` check below, I can remove them